core/module/Dog/dog_modules/dog_module/Shadowlamb/core/SR_Player.php

```diff
@@ -2081,9 +2081,16 @@ public function giveItem(SR_Item $item)
 		return $this->sr4_inventory->addItem($item);
 	}
 
-	public function itemAmountChanged(SR_Item $item, $amount_change, $modify=true)
-	{
-		$this->sr4_inventory->itemAmountChanged($item, $amount_change);
+	public function itemAmountChanged(SR_Item $item, $amount_change, $modify=true, $use_mount=false)
+	{
+        if ($use_mount)
+        {
+            $this->sr4_mount_inv->itemAmountChanged($item, $amount_change);
+        }
+        else
+        {
+            $this->sr4_inventory->itemAmountChanged($item, $amount_change);
+        }
```

Comment:
Mixing tabs and spaces.

Reply:
Hard to adopt gizmore's code style.
I'll fix...